clusterloader2/testing/load/config.yaml

```diff
@@ -897,3 +904,4 @@ steps:
       clusterOOMsTrackerEnabled: {{$ENABLE_CLUSTER_OOMS_TRACKER}}
       restartCountThresholdOverrides: {{YamlQuote $RESTART_COUNT_THRESHOLD_OVERRIDES 4}}
       enableRestartCountCheck: {{$ENABLE_RESTART_COUNT_CHECK}}
+{{end}}
```

Comment:
I'm not sure if this is a good idea to skip gathering measurements. We may want to use those at some point and they should have little to no impact on cluster, so it should be ok to run them after exec command.

Reply:
Done. I've also added comment that indicates which if ends.